torchmetrics/functional/text/bert.py

```diff
@@ -530,14 +539,20 @@ def bert_score(
             A path to the user's own local csv/tsv file with the baseline scale.
         baseline_url:
             A url path to the user's own  csv/tsv file with the baseline scale.
+        predictions:
+            .. deprecated:: v0.7
+                This argument is deprecated in favor of  `preds` and will be removed in v0.8.
+        references:
+            .. deprecated:: v0.7
+                This argument is deprecated in favor of  `preds` and will be removed in v0.8.
```

Comment:
```suggestion
        predictions:
        
            .. deprecated:: v0.7
                This argument is deprecated in favor of  `preds` and will be removed in v0.8.
                
        references:
        
            .. deprecated:: v0.7
                This argument is deprecated in favor of  `preds` and will be removed in v0.8.
                
```
sorry for the misleading suggestion before...